tests/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+
+services:
+
+  cassh-server:
+    build:
+      context: ../server
+      dockerfile: Dockerfile
+    image: nbeguier/cassh-server
```

Comment:
~nbeguier~ leboncoin?

Reply:
We could but might be hard to push upstream